modules/tracking/include/opencv2/tracking/kalman_filters.hpp

```diff
@@ -222,6 +224,47 @@ CV_EXPORTS Ptr<UnscentedKalmanFilter> createUnscentedKalmanFilter( const Unscent
 */
 CV_EXPORTS Ptr<UnscentedKalmanFilter> createAugmentedUnscentedKalmanFilter( const AugmentedUnscentedKalmanFilterParams &params );
 
+
+namespace internal {
```

Comment:
I think it will be better to move it to `tracking/src/<some-hpp-file>`, for example `precomp.hpp`, or some new file.